modules/nlu/src/backend/language-provider.ts

```diff
@@ -308,6 +309,12 @@ export class RemoteLanguageProvider implements LanguageProvider {
     }) // randomly generated words
   }
 
+  async primeLanguageCaches(intentDefs: sdk.NLU.IntentDefinition[], lang: string): Promise<void> {
```

Comment:
Where is this function called ? It used to be called at svm training

Reply:
You're right, I forgot to call it after adding it to the lang-provider.